backend/src/database/repositories/organizationRepository.ts

```diff
@@ -541,58 +598,40 @@ class OrganizationRepository {
       delete data.attributes.syncRemote
     }
 
+    if (manualChange) {
+      const manuallyChanged: string[] = record.manuallyChanged || []
+
+      for (const column of this.ORGANIZATION_UPDATE_COLUMNS) {
+        let changed = false
+        if (column in record && !(column in data)) {
+          // column was manually removed
+          changed = true
```

Comment:
if data doesn't have the field at all on update (a partial update to some other field), wouldn't this cause the non-sent field to be marked as manually removed?